Vungle/src/main/java/com/mopub/mobileads/VungleRouter.java

```diff
@@ -50,7 +52,37 @@ public void onResume(@NonNull final Activity activity) {
     private static VungleRouter sInstance = new VungleRouter();
     private static SDKInitState sInitState = SDKInitState.NOTINITIALIZED;
     private static Map<String, VungleRouterListener> sVungleRouterListeners = new HashMap<>();
```

Comment:
@HuFangshuai Should we also update this listener to have the key of Map as an AdRequest, as we are now supporting multi bidtokens on SDK, its supposed to unblock Pubs to be able to make multiple AdRequests in parallel. We should be able to start the loadAd operations for such parallel Ad requests for same Placement.
We would only need to put limitation on the actual playing/rendering which will still be supported one at a time for Placement.

Reply:
Never mind, ignore the previous comment. I take that back, saw that currently MoPub doesnt support parallel requests for same AdUnitId/Placement
https://developers.mopub.com/publishers/android/rewarded-ad/#known-limitations

Reply:
Update: Shiv confirmed from MoPub that concurrent requests should work for Interstitial and Banners

Reply:
ok, will implement it on ticket: https://vungle.atlassian.net/browse/AND-3337